### PR TITLE
Review delete files

### PR DESCRIPTION
**Pull request by atuls45**

### CODE REVIEW

File: Frontend/src/Components/Debouncing.jsx [hunk 1]
```diff
@@ -1,15 +0,0 @@
-import { useEffect } from 'react'
```
Comment (atuls45) [inline Frontend/src/Components/Debouncing.jsx:1]: Debouncing is not a component, you can write it in other folder named util or helper

File: Frontend/src/Components/Error.jsx [hunk 2]
```diff
@@ -1,11 +0,0 @@
-import error from "../Assets/Advt/error.jpg";
```
Comment (atuls45) [inline Frontend/src/Components/Error.jsx:1]: Try to use png, svg as much as possible

File: Frontend/src/Components/Error404.jsx [hunk 3]
```diff
@@ -1,24 +0,0 @@
-import { Link } from "react-router-dom";
-import errorRoute from "../Assets/Advt/errorRoute.gif";
-import Footer from "./Footer";
-import Navbar from "./Navbar";
-
-const Error404 = () => {
-  return (
-    <>
-       <Navbar />
-    <div className='w-full flex  justify-center bg-background  m-auto'>
```
Comment (atuls45) [inline Frontend/src/Components/Error404.jsx:10]: Do a proper alignment

File: Frontend/src/Components/HomeSearch.jsx [hunk 4]
```diff
@@ -1,153 +0,0 @@
...
-import useDebouce from './Debouncing';
-import { useRef } from 'react';
-
-const HomeSearch = () => {
-   const [searchparam,setSearchparam] = useSearchParams();
-   const [checkInDate,setCheckinDate] = useState(new Date().toJSON().slice(0, 10));
-   const [checkOutDate,setCheckoutDate] = useState(new Date().toJSON().slice(0, 10));
-   const [query,setQuery] = useState("");
-   const [city,setCity] = useState( searchparam.get("city")||"");
-   const [x,setX] = useState("");
```
Comment (atuls45) [inline Frontend/src/Components/HomeSearch.jsx:25]: Give proper naming convention, what x defines

File: README.md [hunk 5]
```diff
@@ -1,89 +0,0 @@
...
-
-<h4>Here user can see all the data like images , rooms available and reserve then:- </h4>
-<img src="Frontend/images/SinglePage.JPG" width="700"/>
-
-6.Payment Page
-
-<h4>On this page user has to fill Personal Details like name, mobile no , cards details, etc :- </h4>
-<img src="Frontend/images/Payment.JPG" width="700"/>
-
-## Team Member
```
Comment (atuls45) [inline README.md:83]: Mention how to start project and use DB

File: Backend/db.json [hunk 6]
```diff
@@ -1,1413 +0,0 @@
...
-        }
-      },
-      "policies": {
-        "Check-in": {
-          "from": "Check-in from 3:00 PM - anytime",
-          "early": "Early check-in subject to availability",
-          "express": "Express check-in",
-          "minAge": "Minimum check-in age - 18"
-        },
-        "Check-out": {
```
[the remark this answers is not in the document]
Comment (Md-Irfan-FullStackDeveloper) [reply Backend/db.json:82]: Done

File: .gitignore [hunk 7]
```diff
@@ -1 +0,0 @@
-.vercel
```
[the remark this answers is not in the document]
Comment (Md-Irfan-FullStackDeveloper) [reply .gitignore:1]: Already added there

File: Backend/src/configs/db.js [hunk 8]
```diff
@@ -1,21 +0,0 @@
-// const mongoose = require("mongoose");
```
[the remark this answers is not in the document]
Comment (Md-Irfan-FullStackDeveloper) [reply Backend/src/configs/db.js:1]: Removed

File: Frontend/src/Components/Debouncing.jsx (hunk 1 above)
Comment (Md-Irfan-FullStackDeveloper) [reply Frontend/src/Components/Debouncing.jsx:1]: Shifted to utils folder

File: Frontend/src/Components/Error404.jsx (hunk 3 above)
Comment (Md-Irfan-FullStackDeveloper) [reply Frontend/src/Components/Error404.jsx:10]: Done

File: Frontend/src/Components/HomeSearch.jsx (hunk 4 above)
Comment (Md-Irfan-FullStackDeveloper) [reply Frontend/src/Components/HomeSearch.jsx:25]: Done

**Comment by Md-Irfan-FullStackDeveloper** [comment]
Work done according to feedback

**Pull request closed without merging**